airflow/contrib/hooks/sagemaker_hook.py

```diff
@@ -232,15 +234,20 @@ def get_conn(self):
 
     def get_log_conn(self):
         """
-        Establish an AWS connection for retrieving logs during training
+        This method is deprecated. Please use `airflow.contrib.hooks.AwsLogsHook.get_conn` instead.
 
         :rtype: CloudWatchLogs.Client
         """
-        config = botocore.config.Config(retries={'max_attempts': 15})
-        return self.get_client_type('logs', config=config)
+        warnings.warn("Method `get_log_conn` has been deprecated. "
+                      "Please use `airflow.contrib.hooks.AwsLogsHook.get_conn` instead.",
+                      category=DeprecationWarning)
+
+        return self.logs_hook.get_conn()
 
     def log_stream(self, log_group, stream_name, start_time=0, skip=0):
         """
+        This method is deprecated. Please use `airflow.contrib.hooks.AwsLogsHook.get_log_events` instead.
```

Comment:
```suggestion
        This method is deprecated. Please use :py:meth:`get_log_events` instead.
```

We should probably also remove the rest of the old docs here.